File: RepoTransBench/target_projects/Java/Go/cyfonly_FLogger/public_tests/flogger_core_public_test.go
package public_tests

import (
	"testing"
)

type DummyLogger struct{}

func (l *DummyLogger) Debug(msg string) {}
func (l *DummyLogger) Fatal(msg string) {}
func (l *DummyLogger) WriteLogWithFile(file string, level int, msg string) {
}

func GetLogger() *DummyLogger {
	return &DummyLogger{}
}

func TestVariousLevelsPublic(t *testing.T) {
	logger := GetLogger()
	logger.Debug("Debugging - public core test!")
	logger.Fatal("This is a public fatal log!")
	logger.WriteLogWithFile("public_logfile", 2, "This is a public WARN log in a special file!")
}

File: RepoTransBench/target_projects/Java/C#/cyfonly_FLogger/public_tests/FLoggerCorePublicTest.cs
using Xunit;

namespace cyfonly_FLogger.Tests.Public
{
    public class FLoggerCorePublicTest
    {
        [Fact]
        public void TestVariousLevelsPublic()
        {
            var logger = cyfonly_FLogger.FLogger.GetInstance();
            // Use different messages and levels compared to original, including more threads
            logger.Debug("Debugging - public core test!");
            logger.Fatal("This is a public fatal log!");
            logger.WriteLog("public_logfile", 2, "This is a public WARN log in a special file!");
        }
    }
}